.github/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs: 
+  build:
+    name: JDK ${{ matrix.java_version }}
+    runs-on: ubuntu-latest
+  strategy:
+    matrix:
+      java_version: [1.8]
+  steps:
+  - name: Checkout
+  - uses: actions/checkout@v2
+  - nmae: Install JDK ${{ matrix.java_version }}
+  - uses: actions/setup-java@v2
+    with:
+    java-version: ${{ matrix.java_version }}
+  - name: Install Android SDK
+    uses: malinskiy/action-android/install-sdk@release/0.1.1
+  - name: Gradle Wrapper Validation
+    uses: gradle/wrapper-validation-action@v1
+  - name: Configure Gradle
+    # Install Gradle Deps
+    run: ./gradlew help
+  - name: Build project
+    run: ./gradlew assemble --stacktrace
+  - name: Run tests
+    run: ./gradlew test --stacktrace
+  - name: Upload snapshot (main only)
+    run: ./gradlew uploadArchives -PSONATYPE_NEXUS_USERNAME="$SONATYPE_NEXUS_USERNAME" -PSONATYPE_NEXUS_PASSWORD="$SONATYPE_NEXUS_PASSWORD"
+    env:
+      SONATYPE_NEXUS_USERNAME: ${{ secrets.SonatypeUsername }}
```

Comment:
are these secrets already setup?

Reply:
Secrets have to be setup in Settings - https://docs.github.com/en/actions/reference/encrypted-secrets

Reply:
i added them. plz confirm the snapshot uploads properly after Actions run the first time